src/main/java/uk/gov/hmcts/reform/civil/controllers/testingsupport/TestingSupportController.java

```diff
@@ -276,4 +275,11 @@ public ResponseEntity<List<HistoricProcessInstanceDto>> getCamundaProcesses(
     private String getSystemUserToken() {
         return userService.getAccessToken(systemUserConfig.getUserName(), systemUserConfig.getPassword());
     }
+
```

Comment:
This method looks like its a duplicate of getSystemUserToken()

Reply:
Yes, made a mistake - remove duplicate method - thank you